### DISCUSSION TITLE
APPARENT solar time or MEAN solar time?

### DISCUSSION THREAD

**Discussion by leonardcj** [Stellarium/stellarium#28] [Q&A] [answered]
Do these entries in  stellarium-remotecontrol.po refer to APPARENT solar time or MEAN solar time (universal time)?

https://en.wikipedia.org/wiki/Solar_time
An apparent solar day can be 20 seconds shorter or 30 seconds longer than a mean solar day

#: index.html:86 tablet7in.html:187
msgid "Solar hour"
msgstr ""

#: index.html:87 tablet7in.html:188
msgid "Solar day"
msgstr ""

#: index.html:88 tablet7in.html:189
msgid "Solar week"
msgstr ""

**Comment by gzotti** [comment, marked as answer]
These are mean time values. 3600, 86400 and 7x86400 SI seconds, respectively.

**Comment by leonardcj** [comment]
Thank you
